src/main/java/uk/gov/ons/ctp/integration/rhsvc/service/impl/CaseServiceImpl.java

```diff
@@ -38,9 +49,22 @@
 
   @Autowired private RespondentDataRepository dataRepo;
   @Autowired private MapperFacade mapperFacade;
+  @Autowired private EventPublisher eventPublisher;
   @Autowired private ProductReference productReference;
 
-  @Autowired private EventPublisher publisher;
+  private BoundMapperFacade<AddressDTO, AddressModified> addressDTOMapperFacade;
+  private BoundMapperFacade<Address, AddressModified> addressMapperFacade;
+
+  /** Constructor */
+  public CaseServiceImpl() {
+
+    MapperFactory mapperFactory = new DefaultMapperFactory.Builder().build();
```

Comment:
Why have a local MapperFacade and not use the application wide one that is auto wired?

Reply:
Used a local BoundMapperFacade constructed from a MapperFactory for supposed performance advantages.  Less code and neater using application wide RHSvcBeanMapper MapperFacade instance. Changed to use this as suggested.